openfisca_parsers/formulas_parsers_2to3.py

```diff
@@ -2981,7 +2981,7 @@ def parse_value(self, node, container = None):
                             unicode(value).encode('utf-8'))
                     item_value = self.parse_value(dict_children[child_index + 2], container = container)
                     child_index += 3
-                    if dict_children[child_index].type == tokens.COMMA:
+                    if (child_index < len(dict_children)) and dict_children[child_index].type == tokens.COMMA:
```

Comment:
@cbenz,

I don't know why this bug appeared now, but it was making the `web-api` tests to crash. I tried to understand the logic to fix it, but maybe I missed something so I could appreciate a review.